python/pylibvw.cc

```diff
@@ -1008,6 +1008,13 @@ py::object ex_get_ccb_explicitly_included_actions(example_ptr ec)
   return varray_to_pylist(label.explicit_included_actions);
 }
 
+size_t ex_get_multilabel_num_labels(example_ptr ec) { return ec->l.multilabels.label_v.size(); }
```

Comment:
It might be simpler to just return a py::list for this one?

Reply:
Sure that makes it a bit simpler